admin_guide/out_of_resource_handling.adoc

```diff
@@ -224,12 +204,66 @@ pods on the node.
 |`BestEffort` pods are not scheduled to the node.
 |===
```

Comment:
so the scheduler does NOT read eviction signals, it reads Node Conditions that are driven based on the configured eviction thresholds.  So for example, if i set an eviction threshold like the following:

eviction-hard is "memory.available<500Mi"

if available memory falls below that value, the Node has a value reported in `Node.Status.Conditions[]` whose `Type` will be `MemoryPressure` and whose `Status` will be `True`.  It's that value on the node object that the scheduler integrates with when making scheduling decisions.